draw_line.c

```diff
@@ -48,13 +51,26 @@ int main(int argc, char* argv[]) {
       height = screen->height-y-2;
    }
 
+   struct xywh borders[4] = {
+	   // Top
+	   { x-1, y-1, width + 2, 1 },
+	   // Bottom
+	   { x-1, y + height + 1, width + 2, 1 },
+	   // Left
+	   { x-1, y, 1, height },
+	   // Right
+	   { x + width + 1, y, 1, height }
+   };
+
+   for (int i = 0; i < 4; ++i) {
+	   w = XCreateSimpleWindow(d, RootWindow(d, s), borders[i].x, borders[i].y, borders[i].width, borders[i].height, 0,
```

Comment:
teraz to może nazwijmy ten plik `draw_border.c`? 😃

Reply:
Tak, miałem to samo zrobić! Kompletnie zapomniałem